bytecode/tests/test_flags.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+import unittest
+from bytecode import Flags, CoFlags, Bytecode, ConcreteBytecode, ConcreteInstr
+
+
+class FlagsTests(unittest.TestCase):
+
+    def test_instantiation(self):
+
+        empty_flag = Flags()
+        self.assertFalse(any(empty_flag._defaults.values()))
+        self.assertFalse(empty_flag._forced)
+
+        flags = Flags(CoFlags.CO_OPTIMIZED + CoFlags.CO_NESTED)
+        for m in CoFlags.__members__:
+            expected = (True
+                        if m in {'CO_OPTIMIZED', 'CO_NESTED'} else
+                        False)
+            self.assertEqual(flags.get_default(m.split('_', 1)[1].lower()),
```

Comment:
I would prefer a class method to get "all flags" than duplicating this split/lower code.